pcsx2/CMakeLists.txt

```diff
@@ -1583,8 +1583,14 @@ endif()
 
 # MacOSX/BSD
 if(UNIX AND NOT Linux)
+	if(${CMAKE_SYSTEM_NAME} MATCHES "FreeBSD" OR ${CMAKE_SYSTEM_NAME} MATCHES "NetBSD")
```

Comment:
Might be a bit easier to do `if(APPLE)` and assume BSD otherwise

Reply:
Right! I've just pushed an update to the patch. Thanks!